raiden_contracts/constants.py

```diff
@@ -1,7 +1,15 @@
 from enum import Enum, IntEnum
+from typing import NewType
+
+# Something like '0.8.0_unlimited' or '0.8.0' depending on limted/unlimited
+FlavoredVersion = NewType('FlavoredVersion', str)
+
+# Something like '0.8.0' before adding '_unlimited'
+PlainVersion = NewType('PlainVersion', str)
 
 # Do not change this, this is handled by bumpversion with .bumpversion_contracts.cfg
-CONTRACTS_VERSION = "0.8.0"
+_CONTRACTS_VERSION = "0.8.0"
```

Comment:
nitpick: Please keep style consistent with raiden. Use `'` and not `"`

Reply:
I'll fix this in this PR.

Reply:
Fixed in this PR.  I guess the same problem exists all over the code base, so I created an issue https://github.com/raiden-network/raiden-contracts/issues/644